package.json

```diff
@@ -16,12 +16,12 @@
     "./browser/static/ably-node.js": "./browser/static/ably-commonjs.js"
   },
   "files": [
-    "ably.d.ts",
+    "src/ably.d.ts",
     "browser/static/**",
-    "callbacks.d.ts",
-    "callbacks.js",
-    "promises.d.ts",
-    "promises.js",
+    "src/callbacks.d.ts",
```

Comment:
Yeah the changes here are breaking changes, I would suggest using webpack copy plugin to these files into the dist folder and change the release process to use `npm publish dist`